teachings.json

```diff
@@ -900,10 +900,37 @@
   {
     "name": "Computer Vision",
     "url": "computer-vision",
-    "chat": "",
+    "chat": "https://t.me/+9wTytT_D4Lg1MjFk",
     "website": "479049",
     "professors": [
       "giuseppe.lisanti"
     ]
+  },
+  {
+    "name": "Artificial Intelligence, Blockchain e criptovalute nello sviluppo software",
+    "url": "artificial-intelligence-blockchain-e-criptovalute-nello-sviluppo-software",
+    "chat": "@AIBC_unibo",

```

Comment:
I am pretty sure a full url is required. As groups on different instant messaging apps are supported.

Reply:
> I am pretty sure a full url is required. As groups on different instant messaging apps are supported.

Sorry! My mistake! I will patch it up soon. Then, if there are no problems, I think we could pass this PR so at least we have some new pages for courses that hust started.

Reply:
> I am pretty sure a full url is required. As groups on different instant messaging apps are supported.

Sorry! My mistake! I will patch it up soon. Then, if there are no problems, I think we could pass this PR so at least we have some new pages for courses that just started.

Reply:
Yes, will merge is as soon as the path is pushed. After that happens, we must also merge dependabot's PR to update this dependency (csunibo/config) within the other projects (e.g., dynamik and informabot)